testdata/summary_test.go

```diff
@@ -0,0 +1,98 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	parser "github.com/babylon/parsers/nmparser"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSummary(t *testing.T) {
+	var tests = []struct {
+		modFile    string
+		goldenFile string
+	}{
+		{
+			modFile:    "./example-models/nonmem/BQL/2.mod",
+			goldenFile: "./example-models/nonmem/BQL/BQL.json",
+		},
+		{
+			modFile:    "./example-models/nonmem/IOVMM/10.mod",
+			goldenFile: "./example-models/nonmem/IOVMM/IOVMM.json",
+		},
+		{
+			modFile:    "./example-models/nonmem/TMDD/1.mod",
+			goldenFile: "./example-models/nonmem/TMDD/TMDD.json",
+		},
+		{
+			modFile:    "./example-models/nonmem/NonLinearCL/1.mod",
+			goldenFile: "./example-models/nonmem/NonLinearCL/NonLinearCL.json",
+		},
+	}
+	bbiExe := "/usr/local/bin/bbi"
```

Comment:
this should detect bbi on the path rather than hard coded.

Reply:
ok, but at some point I think we need to be clear about: (1) tools being installed correctly, and (2) executing the tool from the installed folder by default.  It's easier to support a tool that includes its own installer and users should only expect support when they have they have used the installer - which presumably installs the latest supported version to a known location. That's not to say the tool can only be installed in a certain location, but we can start with a recommendation.  BTW, in the krn\csh shell scripts I've written in the past, it was always risky to rely on the users PATH to get the correct version.